test/CMakeLists.txt

```diff
@@ -79,5 +79,4 @@ macro (add_subdirectories)
         endif ()
     endforeach ()
 endmacro ()
-
```

Comment:
The editor Atom inserted it automatically (and I even cannot remove it :( ). I was not aware of it. Since we require a newline for C-code files, we may allow it here, too? Or is there are C/C++-compiler-specific reason for newlines at the end?